nnvm/python/nnvm/frontend/darknet.py

```diff
@@ -267,6 +267,13 @@ def _darknet_upsampling(inputs, attrs):
     new_attrs['scale'] = attrs.get('scale', 1)
     return _darknet_get_nnvm_op(op_name)(*inputs, **new_attrs), None
 
+def _darknet_l2normalize(inputs, attrs):
+    """Process the l2 normalization operation."""
+    op_name, new_attrs = 'l2_normalize', {}
+    new_attrs['eps'] = attrs.get('eps', 0)
+    new_attrs['axis'] = attrs.get('axis', 1)
```

Comment:
I don't really know darknet, would appreciate if you could point out where these two attrs come from. The overall functionality looks good to me.

Reply:
Sure. The darknet computes the l2 normalization always along the channel axis. So it doesn't store the axis information anywhere. And while dividing by sum it doesn't check if sum is greater than eps or not. 
So i have made default eps as 0 and axis as 1 to make it compatible with TVM's L2 normalization implementation. You can have look at darknet's l2 normalization implementation [here](https://github.com/pjreddie/darknet/blob/61c9d02ec461e30d55762ec7669d6a1d3c356fb2/src/blas.c#L126)

Reply:
got it. thanks!